### PR TITLE
[stable/insights-agent] Added cloud costs plugin

### PR DESCRIPTION
**Pull request by jdesouza**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

File: stable/insights-agent/templates/cloud-costs/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                secretKeyRef:
+                  name: {{ .Values.cloudcosts.secretName }}
+                  key: AWS_SECRET_ACCESS_KEY
+            - name: AWS_DEFAULT_REGION
+              value: {{ required "You must set cloudcosts.region, please see https://insights.docs.fairwinds.com/technical-details/reports/aws-costs/#agent-configuration" .Values.cloudcosts.region }}
+            {{ include "proxy-env-spec" . | indent 12 | trim }}
+            command: ["./cloud-costs.sh"]
+            args:
+              - --database
+              - {{ required "You must set cloudcosts.database, , please see https://insights.docs.fairwinds.com/technical-details/reports/aws-costs/#agent-configuration" .Values.cloudcosts.database }}
```
Comment (vitorvezani) [inline stable/insights-agent/templates/cloud-costs/cronjob.yaml]:
```suggestion
              - {{ required "You must set cloudcosts.database, please see https://insights.docs.fairwinds.com/technical-details/reports/aws-costs/#agent-configuration" .Values.cloudcosts.database }}
```

File: stable/insights-agent/ci/test-values.yaml [hunk 2]
```diff
@@ -149,6 +149,19 @@ awscosts:
   tagvalue: staging.internal.reactiveops.com
   workgroup: cur_athena_workgroup
 
+cloudcosts:
```
Comment (vitorvezani) [inline stable/insights-agent/ci/test-values.yaml:152]: I wonder if we want to have a provider specific block. i.e.: for `aws`
```
cloudcosts:
  enabled: true
  aws:
    accessKey: xyz
    secretAccess: xyz
    ....
```

**Review by vitorvezani — APPROVED** [review]

File: stable/insights-agent/ci/test-values.yaml (hunk 2 above)
Comment (jdesouza) [reply stable/insights-agent/ci/test-values.yaml:152]: yeah that makes sense

File: stable/insights-agent/ci/test-values.yaml [hunk 3]
```diff
@@ -149,6 +149,20 @@ awscosts:
...
+  enabled: false
+  secretName: cloudcosts-secret
+  tagkey: kubernetes_cluster
+  tagvalue: staging.internal.reactiveops.com
+  aws:
+    awsAccessKeyId: someAccessID
```
Comment (vitorvezani) [inline stable/insights-agent/ci/test-values.yaml]: You don't need `aws` prefixing here due to context

Comment (jdesouza) [reply stable/insights-agent/ci/test-values.yaml]: right

Comment (jdesouza) [reply stable/insights-agent/ci/test-values.yaml]: done

**Pull request merged**